src/wp-includes/rss.php

```diff
@@ -63,7 +63,8 @@ function __construct( $source ) {
 		# Check if PHP xml isn't compiled
 		#
 		if ( ! function_exists('xml_parser_create') ) {
-			return trigger_error( "PHP's XML extension is not available. Please contact your hosting provider to enable PHP's XML extension." );
+			wp_trigger_error( __METHOD__, "PHP's XML extension is not available. Please contact your hosting provider to enable PHP's XML extension." );
```

Comment:
Thinking the constructor method within this error message is not adding value and may confuse.
```suggestion
			wp_trigger_error( '', "PHP's XML extension is not available. Please contact your hosting provider to enable PHP's XML extension." );
```